provisioning/service/src/provisioningserviceclient.ts

```diff
@@ -269,7 +269,7 @@ export class ProvisioningServiceClient {
   }
 
   private _versionQueryString(): string {
-    return '?api-version=2018-04-01';
+    return '?api-version=2018-09-01-preview';
```

Comment:
>?api-version=2018-09-01-preview'; [](start = 12, length = 33)

i've updated the dps-preview branch so you can actually remove those changes and submit the PR against the dps-preview branch.